Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -150,6 +150,32 @@ function rotateData(obj,axis,value)
             obj.rotateTable(obj.forces, axis, value);
             disp('Marker and Force tables have been rotated')
         end
+        function convertMillimeters2Meters(obj)
+            % Function to convert displacement forceplate measurements made
+            % in millimeters to meters. This will convert point data (mm)
+            % to m and Torque data (Nmm) to Nm.
+            import org.opensim.modeling.*
+            
+            nRows  = obj.forces.getNumRows();
+            labels = obj.forces.getColumnLabels();
+            
+            for i = 0 : obj.forces.getNumColumns - 1
+                % All force columns will have the 'f' prefix while point
+                % and moment columns will have 'p' and 'm' prefixes,
+                % respectively. 
+                if ~contains(char(labels.get(i)),'f')
+                    for u = 0 : nRows - 1
+                        % Get the Vec3
+                        vec3 = obj.forces.getDependentColumnAtIndex(i).get(u);
+                        % Divide the Vec3 by 1000
+                        vec3_new = Vec3(vec3.get(0)/1000,vec3.get(1)/1000,vec3.get(2)/1000);
+                        % set the table value
```

Comment:
This is excessively expensive to divide a Vec3 by a double, I''m looking into exposing the equivalent of operator for +=,-=, *=, /= to Java/Matlab so avoid the inefficiency/overhead but looks fine considering what we have now.

Reply:
PR #2558 should allow you to replace this with `vec3.scalarDivideEq(1000)` in a fraction of the time but don't hold this PR off for it.